### DISCUSSION TITLE
Why is the initial value of `$env.CMD_DURATION_MS` set to `0823`?

### DISCUSSION THREAD

**Discussion by lewis-yeung** [nushell/nushell#6402] [Q&A] [answered]
@fdncred When I open a new Nushell session, the initial value of `$env.CMD_DURATION_MS` is always `0823`. Since it reflects the command execution time, shouldn't the initial value be `0` instead?

Related commit: 2df37d6.

**Comment by fdncred** [comment, marked as answer]
08/23 is the day nushell was first released. so, it's an "easter egg".

**Reply by lewis-yeung** [reply]
Aha, that makes sense. But how can I set the initial value to `0` manually? Is it possible in a config file?

**Reply by fdncred** [reply]
you cannot. it's not exposed to config or anything else. i guess you could maybe do `let-env CMD_DURATION_MS = "0"` in one of your config files?

**Reply by lewis-yeung** [reply]
Yes, but it does not work.

**Reply by fdncred** [reply]
ya you're right. The code would have to be changed to allow that, which I'm not inclined to do atm.
